base/FileMetadataUtilities.F90

```diff
@@ -61,10 +61,12 @@ function var_get_missing_value(this,var_name,rc) result(missing_value)
       integer, optional, intent(out) :: rc
 
       integer :: status
+      character(len=ESMF_MAXSTR) :: fname
       type(Variable), pointer :: var
 
+      fname = get_file_name(this,_RC)
       var => this%get_variable(var_name,_RC)
-      _ASSERT(associated(var),"no variable named "//var_name//" in file")
+      _ASSERT(associated(var),"no variable named "//var_name//" in "//fname)
```

Comment:
@lizziel Can you try this pattern instead in here:

```fortran
character(:), allocatable :: fname
...
fname = this%get_file_name(_RC)
```

Reply:
Yes, the change was fine for GCHP. I applied it to all instances in the file.